src/main/java/org/traccar/BasePipelineFactory.java

```diff
@@ -158,8 +158,8 @@ protected void initChannel(Channel channel) {
                 GeocoderHandler.class,
                 SpeedLimitHandler.class,
                 MotionHandler.class,
-                CopyAttributesHandler.class,
                 EngineHoursHandler.class,
+                CopyAttributesHandler.class,               
```

Comment:
What is going on here again?

Reply:
Sorry let me correct this. Reordered incorrectly . Need to move the enginehour handler under computed attributes handler so calculated ignitions also work with hours

Reply:
This does not belong in this pull request.